Terminal.Gui/Views/TextField.cs

```diff
@@ -253,6 +250,14 @@ public override void Redraw (Rect bounds)
 			PositionCursor ();
 		}
 
+		Attribute GetReadOnlyColor ()
+		{
+			if (ColorScheme.Disabled.Foreground == ColorScheme.Focus.Background) {
```

Comment:
This check for Disabled foreground matching the background is only in TextField and not the other views... is there a reason why this only applies to `TextField`?

Reply:
Yes there are. The background color of the Disabled attribute is the same as the background color of the Normal attribute, but for the TextField we need to draw the box, so this is the reason.